src/test/java/rose/ParserTest.java

```diff
@@ -0,0 +1,48 @@
+package rose;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
I think static imports should be ordered first. Check the ordering of imports [here](https://se-education.org/guides/tutorials/intellijCodeStyle.html).